pkg/mocks.go

```diff
@@ -1,7 +1,16 @@
-package main
+package pkg
 
 import "net/http"
 
+// HTTPClient interface
+type HTTPClient interface {
+	Do(req *http.Request) (*http.Response, error)
```

Comment:
We usually `make` or `send` a request.

Reply:
It is part of the original API 😬

https://golang.org/src/net/http/client.go?s=20406:20458#L574

Reply:
Got it.